### PR TITLE
Selenium fixes

### PR DESCRIPTION
**Pull request by bjones1**
This prints out npm run build output, in addition to a bit of updating.

### CODE REVIEW

**Comment by bjones1** [comment]
The tests now pass, which *may* mean one of the Selenium timing problems are fixed.

File: runestone/mchoice/test/test_assess.py [hunk 1]
```diff
@@ -1,16 +1,37 @@
...
 
 from unittest import TestCase
-from selenium.webdriver.support import expected_conditions as EC
 from selenium.webdriver.common.by import By
 from runestone.unittest_base import module_fixture_maker, RunestoneTestCase
 
 mf, setUpModule, tearDownModule = module_fixture_maker(__file__, True)
 
+
+class element_has_css_class:
```
Comment (bnmnetp) [inline runestone/mchoice/test/test_assess.py:14]: Ooooh nice!

Comment (bnmnetp) [reply runestone/mchoice/test/test_assess.py:14]: Actually this seems like it would be good to have in unittest_base so that we can use it in other tests

Comment (bjones1) [reply runestone/mchoice/test/test_assess.py:15]: Good point. I'll move it there.

**Comment by bjones1** [comment]
Sigh, a test failure popped up and even the 5 second wait didn't help. I'll test locally to see if I can reproduce this.

**Pull request merged**